### PR TITLE
*: cut v0.7.0-rc.0

### PR DESCRIPTION
**Pull request by mxinden**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## v0.7.0-rc.0 / 2017-05-19
+
+* [BUGFIX] Enable builds for Solaris/Illumos
```
Comment (grobie) [inline CHANGELOG.md]: We usually sort things the other way around: CHANGE, FEATURE, ENHANCEMENT, BUGFIX. See prometheus/prometheus for examples.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+* [ENHANCEMENT] Rewrite UI from scratch improving UX
```
Comment (grobie) [inline CHANGELOG.md]: I'd label this as `[CHANGE]` as it's so big.

File: CHANGELOG.md (hunk 1 above)
Comment (mxinden) [inline CHANGELOG.md]: Oh, sorry. Changed.

File: CHANGELOG.md (hunk 2 above)
Comment (mxinden) [inline CHANGELOG.md]: Ok, changed as well. Thanks for the feedback!

**Comment by brancz** [comment]
lgtm :ship: :sparkles:

**Pull request merged**